index.js

```diff
@@ -0,0 +1,87 @@
+module.exports = {
+  parser: "@typescript-eslint/parser",
+
+  extends: ["plugin:react/recommended", "prettier/@typescript-eslint", "plugin:prettier/recommended", "prettier/react"],
+
+  plugins: ["simple-import-sort", "jest", "@typescript-eslint", "react-hooks", "react", "prettier", "import"],
+
+  ignorePatterns: ["coverage", "**/dist", ".cache-loader", "node_modules", ".yalc"],
+
+  parserOptions: {
```

Comment:
This is unlikely to be needed. IIRC it only applies to the default parser.

Reply:
They apply to whatever parser you're using - and the options depend on parser being used. See https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/parser#configuration for the typescript parser options.